.github/workflows/e2e.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
        # should be compatible with them.
        kube-version:
        - "1.19"
-       - "1.24"
+       - "1.25"
```

Comment:
Question: why do we just use `1.19` and `1.25`? I see there's a 1.19 through 1.25

Reply:
I can change it if you want. I don't have a strong opinion related to this

Reply:
We started doing it on the jaeger-operator because of the time it took to run builds on all supported kubernetes versions.  PRs were backing up for a long time because of this.

We could try running against all supported versions here, but we may run into the same problem.

Reply:
Oh, sorry. I misunderstood the @jaronoff97 comment.
As @kevinearls says, this is done to reduce the time spent in the E2E test executed by the CI. There is a comment in this file about this decision:
```yaml
       # The e2e tests are run on the lowest and highest supported k8s version.
       # All Kubernetes version in between expose the same APIs, hence the operator
       # should be compatible with them.
```

Reply:
Ah thank you for the clarification!